open-company/open-company-web
# Manage members tweaks

Card: https://trello.com/c/I6bMPDxg

To test:
- open user dropdown menu
- [ ] do you see Manage Team? Good
- [ ] does it has the people icon? Good
- click it
- [ ] does it open the Manage Team panel? Good
- [ ] make sure the name is *Manage Team* and not ~Manage Memebers~
- close the settings panel
- now look at the links in bottom left corner
- [ ] do you see Support below Invite People? Good
- [ ] do you see the chat bubbles icon for Support? Good
- [ ] do you see the plus icon for Invite People? Good
